issue35 add total assigned hour,deleted unused files

yylocky opened the pull request:
## Change Summary 📝
[Briefly summarise the changes that you made.]
1. Added total assigned hours calculation on view page for the allocation records.
![image](https://github.com/yylocky/CITS5206-Group3/assets/99595619/daa93c6f-09e2-4a6a-8c2a-2a0515e02efc)
2. Adjusted exported csv file accordingly.
![image](https://github.com/yylocky/CITS5206-Group3/assets/99595619/e16e23bf-25e1-4cd9-99d9-e77ad61b60f4)
3. Deleted files that haven't been used.
---

## Change Form 📋
Please mark as NA if not applicable. If a certain criterion is not met, kindly provide a reason.

- [x] The pull request title has an issue number.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] Relevant tests (if applicable) have been added or updated.

---

## Any background context you want to provide? 🖼️

Any additional context or screenshots that could help the reviewers?


## Reviewers 🕵️

(Optional) Suggest reviewers that might be interested in this PR.

yylocky committed:
add total assigned hour,deleted unused files

kylelhk reviewed (commented):
Minor suggestions:
1.	The test data circled below is wrong:
<img width="524" alt="image" src="https://github.com/yylocky/CITS5206-Group3/assets/116782485/ecbf05f8-e5d3-49c0-b46e-57d0031fb7fd">

2.	Assigned Hours and Workload Point – all in 2 decimal places

Open issue to be documented:
- To account for special characters (e.g. comma, hash) that can interrupt the CSV generation logic (refer to issue 36)

yylocky committed:
fix bug on exportcsv function, correct test data, add special characters in test db for testing

yylocky committed:
fix dispay format for total hours for simplicity

yylocky commented:
@kylelhk  Thanks for spotting the problems 👍🏻 
![image](https://github.com/yylocky/CITS5206-Group3/assets/99595619/aefaa689-d0da-48b4-b7ba-0da3398aa967)

1. Corrected test data.
2. accounted for special characters when exporting csv
![image](https://github.com/yylocky/CITS5206-Group3/assets/99595619/0c724109-d737-4fe0-9f86-73e7c7d0f1f4)

3. changed display format for total hour, for simplicity and consistency, now all the number will be displayed "as-is" from db, not necessarily fixed decimal point, this is for now the optimisation I think worth doing for the sake of time use.

kylelhk reviewed (approved):
Looks good.  The display of decimal place is a minor one.
